bundles/org.openhab.binding.nikobus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -35,14 +36,63 @@
 		</channels>
 
 		<config-description>
-			<parameter name="address" type="text">
+			<parameter-group name="parameterGroupPC">
+				<label>Parameters retrieved from the Niko PC application</label>
+				<description>Parameters retrieved from the Niko PC application</description>
+				<advanced>false</advanced>
+			</parameter-group>
+			<parameter-group name="parameterGroupBus">
+				<label>Parameters retrieved from the bus</label>
+				<description>String</description>
+				<advanced>true</advanced>
+			</parameter-group>
+
+			<parameter name="addressPC" type="text"
+				groupName="parameterGroupPC" pattern="([0-9a-fA-F]){6}">
+				<label>Address</label>
+				<description>The Nikobus address of the module</description>
+			</parameter>
+			<parameter name="addressButtonPC" type="text"
+				groupName="parameterGroupPC">
+				<label>Specific Button</label>
+				<description>The specific button</description>
+				<options>
+					<option value="2_1">2-button A (top)</option>
+					<option value="2_3">2-button B (bottom)</option>
+					<option value="4_1">4-button A (top left)</option>
+					<option value="4_3">4-button B (bottom left)</option>
+					<option value="4_0">4-button C (top right)</option>
+					<option value="4_2">4-button D (bottom right)</option>
+					<option value="8_5">8-button 1A (top left)</option>
```

Comment:
To be honest, this does not feel very natural - if the goal is to have same config in Nikobus PC software and in OH, than I would suggest to use 1:1 mapping - please note many people still use text configurations (and not i.e. PaperUI) and at least I would need to check what `8_5` actually maps to.

That said I would suggest to use something similar to `"<address>:<number of buttons>:<button>"`

this way we are using the well established OH config format using : as separator and unify the address definition for modules and buttons (no need for the additional `addressButtonPC` parameter).

`addressPC="13F588:8:1A"`

Of course, mapping needs to be implemented - so 1A -> 5 in this case. wdyt?

Reply:
Is number of buttons actually needed?

Reply:
I agree on the format, addressPC="13F588:8:1A" is more natural (I like PaperUI but understand most people seem to be using the text config). 
I added the number of buttons to the value because the HTML dropdown list in PaperUI didn't like to have multiple entries with the same value.

Reply:
I'm going with 13F588:1A now and have removed the drop down list

Reply:
done